front/app/modules/commercial/id_bogus/components/VerificationFormBogus.tsx

```diff
@@ -24,6 +24,10 @@ import useAuthUser from 'hooks/useAuthUser';
 // services
 import { verifyBogus } from '../services/verify';
 
+// api
+import { queryClient } from 'utils/cl-react-query/queryClient';
```

Comment:
Do you think we should use the hook instead of importing the queryClient directly? Could be better if it comes from the React context vs this global import.